spec/services/proofing/lexis_nexis/ddp/verification_request_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe Proofing::LexisNexis::Ddp::VerificationRequest do
+  let(:dob) { '01/01/1980' }
```

Comment:
Related to @mitchellhenke 's comment earlier... `Date.parse` no longer handles the "american style" date we used to use... this is an ambigous format with possibly unexepected results:

```ruby
Date.parse('13/12/1970')
=> #<Date: 1970-12-13 ((2440934j,0s,0n),+0s,2299161j)>
```

so we should use the unambiguous ISO format instead:
```suggestion
  let(:dob) { '1980-01-01' }
```

(this is why we added `DateHelper.parse_legacy` in #5191 and #5167)

Reply:
Nice